packages/jsapi-components/src/index.ts

```diff
@@ -0,0 +1,5 @@
+export { default as DateInput } from './DateInput';
```

Comment:
This conflicts with your other change, I think just rebase and you don't need `DateInput` in `jsapi-components`

Reply:
No it's in `@deephaven/components` now, doesn't need the JSAPI since you made those changes

Reply:
jsapi-components/src/DateInput.jsx should also be deleted

Reply:
You're right, I'll clean it up.